tests/asgi/tests.py

```diff
@@ -104,7 +105,8 @@ async def test_file_response(self):
             "django.contrib.staticfiles.finders.FileSystemFinder",
         ],
     )
-    async def test_static_file_response(self):
+    @mock.patch("tempfile.SpooledTemporaryFile.close")
+    async def test_static_file_response(self, close_stream_mock):
```

Comment:
The `patch` here means that the underlying `close()` isn't performed, resulting in the... 

```
ResourceWarning: Unclosed file <tempfile.SpooledTemporaryFile object at 0x...>
```

... on Python 3.11. 

I'm not sure if it's worth checking — If it's not called, we'll get the `ResourceWarning`. 🤔

@lundberg Can you test against the 3.11 beta in order to check this yourself?